compiler/optimizing/iv_simplifier.cc

```diff
@@ -196,7 +196,7 @@ void InductionVarSimplification::PerformReduction(HLoopInformation* loop, HInstr
     new_phi->AddInput(new_add);
     HConstant* biv_inc_cst = biv_increment_->GetConstantRight();
     // Ensured in IsCandidatePhi()
-    DCHECK(biv_inc_cst != nullptr);
+    CHECK(biv_inc_cst != nullptr);
     HConstant* biv_cst = GetNewConstant(HInstruction::InstructionKind::kMul, biv_inc_cst, std::get<1>(val));
     DCHECK(biv_cst != nullptr);
```

Comment:
I can see one more "DCHECK" below for "biv_cst". Was issue only regarding "biv_inc_cst" and not "biv_cst" ?